### DISCUSSION TITLE
Jyers on SKR boards

### DISCUSSION THREAD

**Discussion by andrey9492** [Jyers/Marlin#502] [Q&A] [answered]
Has anyone managed to make this fw work with and SKR board? Recently upgraded to SKR 1.4 turbo with TMC2209 for sensorless homing, linear advance and other goodies.
Managed to get everything to work with vanilla Marlin + TFT35 display but i really miss firmware in combination with the dwin display.

**Comment by Jyers** [comment, marked as answer]
If you compile your own version it should be relatively easy. Just check the wiki for the settings required to get the screen working and add them to your config.

**Reply by andrey9492** [reply]
Hello,
Thanks for the response, that was the first thin i tried, but i tried compiling with the most basic changes to config:
#define MOTHERBOARD BOARD_CREALITY_V4
to
#define MOTHERBOARD BOARD_BTT_SKR_V1_4_TURBO

and 
#define X_DRIVER_TYPE TMC2208_STANDALONE
to
#define X_DRIVER_TYPE TMC2209

i get some missing files errors :
![image](https://user-images.githubusercontent.com/31762207/114934070-173f8100-9e42-11eb-8a78-870d3c42df9b.png)

The weird part is this happens even if i only change the board. I tried fixing the errors by adding additional "../" to the paths it's nagging about but in the end i hit this :
![image](https://user-images.githubusercontent.com/31762207/114934524-a8165c80-9e42-11eb-8016-75b6d7226d49.png)

**Reply by Jyers** [reply]
I'd try starting with a fresh download of the repo as it looks like something has gone a bit funky there.

**Reply by andrey9492** [reply]
This is with a fresh download of the repo, i can't understand why changing the board would lead to not being able to include files as written in the code. 
Will try to debug it further, maybe there is some wierd interaction with LPC1769 and extui.
